server/datastore/mysql/vpp.go

```diff
@@ -28,6 +28,7 @@ SELECT
 	vap.name,
 	vap.latest_version,
 	vat.self_service,
+	vat.id vpp_apps_teams_id,
```

Comment:
TIL you don't need `AS`. Would it be worth adding just to line up with other queries?

Reply:
fair enough, yeah good call to bring to BE sync!